test/ode/rkc_tests.jl

```diff
@@ -19,6 +19,18 @@ Random.seed!(123)
     eigest = integrator.eigen_est
     @test eigest ≈ eigm rtol=0.1eigm
   end
+
+  for i in 1:10, iip in [true, false]
```

Comment:
Just do `for i in 1:10, iip in [true, false], alg in [ROCK2(), ROCK4()]` and `integrator = init(prob, ROCK4())`